corehq/apps/reminders/models.py

```diff
@@ -1484,7 +1488,13 @@ def _recipient_lookup(self):
         elif handler.recipient == RECIPIENT_SURVEY_SAMPLE:
             return CommCareCaseGroup.get(handler.sample_id)
         elif handler.recipient == RECIPIENT_OWNER:
-            return get_wrapped_owner(get_owner_id(self.case))
+            owner_id = get_owner_id(self.case)
+
+            location = SQLLocation.by_location_id(owner_id)
+            if location:
+                return [location]
```

Comment:
@czue it's to match the output of https://github.com/dimagi/commcare-hq/blob/533cb0639e401676bcc5f434fa52de46844cc1d9/corehq/apps/reminders/models.py#L1247